library/ssl_srv.c

```diff
@@ -4213,7 +4213,7 @@ static int ssl_in_client_key_exchange_parse( mbedtls_ssl_context *ssl,
                                           unsigned char *buf,
                                           size_t buflen )
 {
-    int ret;
+    int ret = MBEDTLS_ERR_SSL_INTERNAL_ERROR;
```

Comment:
Why did you choose `INTERNAL_ERROR` and not` FAULT_ERROR`?

Reply:
This was the error that was used in other FI countermeasures for ssl as well.

Reply:
After another inspection, INTERNAL_ERROR was later refactored to `MBEDTLS_ERR_PLATFORM_FAULT_DETECTED` - you're right :) I'll change that.

Reply:
Fixed.